### DISCUSSION TITLE
Automatically add gamelist.xml to config files

### DISCUSSION THREAD

**Discussion by THFourteen** [OnionUI/Onion#515] [Ideas]
It would be great if there was an option to add the below line to each of the config files by default (example for SFC)
"gamelist":"../../Roms/SFC/gamelist.xml",

**Comment by dwmccoy** [comment]
Please refer to the FAQ [On Gamelist](https://github.com/jimgraygit/Onion/wiki/3.-FAQ-(Frequently-asked-questions)#what-is-the-purpose-of-game-lists-and-how-can-i-make-one-in-onion-) and how to use a  miyoogamelist.xml file to answer your suggestion.

**Reply by THFourteen** [reply]
Hey sorry - I think you misunderstood me, I know how to use it (also I meant for other emulators, not arcade ones).

The idea i have is, whenever you upgrade the onion OS software, it overwrites all the config.json files for each emulator. So you have to edit them every time and add the line for gamelist back into the json. It would be good to have the line there already.

Also the extension lists in the json are blank, which causes problems if you have a mix of games like zip and gba in the same folder. Maybe worth adding in some default extensions like this? : 
"extlist":"zip|gba|ZIP|GBA"

cheers
